AdobeStockImageAdminUi/Test/Mftf/Section/AdminAdobeStockConfigurationSection.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="AdminAdobeStockConfigurationSection">
```

Comment:
Ideally, we should use `magento:config:set` for setting the configuration values. That's the main reason, why in most of the tests you cannot find the action group for going to the config page and filling the values.
Please, check the following example
https://github.com/magento/magento-functional-tests-migration/blob/0195dd8f45d947c9d6316ea550ee28714a6f7e79/app/code/Magento/Backend/Test/Mftf/Test/AdminLoginAfterJSMinificationTest.xml#L26 which uses the following data https://github.com/magento/magento-functional-tests-migration/blob/bb5df5108196692245d74a7c75fef3f34f437982/app/code/Magento/Store/Test/Mftf/Data/StoreConfigData.xml#L24
Using `magento:config:set` is a prefered way for setting configuration since it's much faster in comparison with doing everything via browser. The only exception is a test that tests the configuration forms behavior.

Reply:
Implemented